group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/CurrentAssignmentBuilderTest.java

```diff
@@ -19,16 +19,9 @@
 import org.apache.kafka.common.Uuid;
```

Comment:
All the tests have been fully replaced in this file so we can just ignore the diff.